docs/docs/cmd/spo/listitem/listitem-batch-add.mdx

```diff
@@ -13,12 +13,15 @@ m365 spo listitem batch add [options]
 ## Options
 
 ```md definition-list
-`-p, --filePath <filePath>`
-: The absolute or relative path to a flat file containing the list items.
-
 `-u, --webUrl <webUrl>`
 : URL of the site.
 
+`-p, --filePath [filePath]`
+: The absolute or relative path to a flat file containing the list items.
+
+`-c, --csvContent [csvContent]`
+: A string content in CSV-format containing the list items. Specify filePath or csvContent, but not both.
+
 `-l, --listId [listId]`
 : ID of the list. Specify either `listTitle`, `listId` or `listUrl`, but not multiple.
 
```

Comment:
Let's also add a new example with this new option. As mentioned initially, when we get this command working in unison with `ConvertTo-CSV`. Let's add a remark about which params are required in `ConvertTo-CSV` to make sure this command works, if needed.